### PR TITLE
Use proper Groovy logging in J2objcUtils.

### LINKED ISSUE #134: No such property: logger for class: com.github.j2objccontrib.j2objcgradle.tasks.J2objcUtils
When running the j2objcTranslate task I get: 

```
Execution failed for task ':j2objcTranslate'.
> No such property: logger for class: com.github.j2objccontrib.j2objcgradle.tasks.J2objcUtils
```

### PR DESCRIPTION
**Pull request by advayDev1**
Fixes #134

### CODE REVIEW

File: src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/J2objcUtils.groovy [hunk 1]
```diff
@@ -15,6 +15,8 @@
  */
 
 package com.github.j2objccontrib.j2objcgradle.tasks
+
+import groovy.util.logging.Slf4j
```
Comment (brunobowden) [inline src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/J2objcUtils.groovy]: Add note on why this file logs differently than the others

**Comment by brunobowden** [comment]
LGTM after you've add the note

Once you've committed, I'll update the version, tag and publish it

**Comment by advayDev1** [comment]
Sgtm

**Comment by brunobowden** [comment]
@advayDev1 - please go ahead with the merge and I'll do a spin

**Comment by advayDev1** [comment]
done, merging

**Pull request merged**